### DISCUSSION TITLE
Customer Portal - Let user choose language of page

### DISCUSSION THREAD

**Discussion by suther** [kimai/Paid-plugins#194] [Questions & Answers] [answered]
Is there any way that the user can choose the language of the customer portal? 
I think something like a pulldown-menu in the page would be nice... or a language-variable in the URL like `?lang=fr`

Or does it currently automatically set the language of the browser?

In my case I have a frech customer but the customer portal overview is shown in german language.
![image](https://github.com/user-attachments/assets/e8400a7c-bfcb-4990-864d-5f7c97f6fa5b)

**Comment by suther** [comment, marked as answer]
Found a workaround... If I change the URL to `https://mydomain.com/de/auth/customer-portal`.... to`https://mydomain.com/en/auth/customer-portal...` it's in english. Thats ok for my needs.

**Reply by kevinpapst** [reply]
That's actually also the recommended way how to do it. 
But I will add an explanation in the UI and/or add some documentation for it.

**Reply by kevinpapst** [reply]
I updated the documentation: https://www.kimai.org/documentation/plugin-customer-portal.html#changing-the-language

Next plugin update will bring the in-place link to the documentation, which I forgot to add initially.
